content/documentation/device-examples/components/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Components
+weight: 50
+---
+
+The core Pi4J V.2 library doesn't contain any specific support for devices like buttons, motors, LCD etc. This page lists widely used components
+for which a Java implementation, based on Pi4J V.2, is available.
+
+### Current available components
+
+| Device(s)                     | Developed by          | Link              |
+| :---                          | :---                  | :---              |
+| [Simple Button](/documentation/device-examples/simplebutton) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
```

Comment:
maybe we want to take a link directly to the class, once it is merged in main? Could look like: https://github.com/Pi4J/pi4j-example-components/blob/Dev-Arcade/src/main/java/com/pi4j/components/components/SimpleButton.java or https://github.com/Pi4J/pi4j-example-components/blob/main/src/main/java/com/pi4j/components/components/SimpleButton.java

Reply:
i will ask this during the next meeting